packages/contracts-bedrock/src/libraries/Predeploys.sol

```diff
@@ -90,7 +90,7 @@ library Predeploys {
     address internal constant L2_TO_L2_CROSS_DOMAIN_MESSENGER = 0x4200000000000000000000000000000000000023;
 
     /// @notice Returns the name of the predeploy at the given address.
-    function getName(address _addr) internal pure returns (string memory out_) {
+    function getName(address _addr, bool _useInterop) internal pure returns (string memory out_) {
```

Comment:
you can update the function signature to remove the unused arg

Reply:
ok decided to move the change introducing the argument to https://github.com/ethereum-optimism/optimism/pull/10417